### ISSUE TITLE
background and foreground colors have insufficient contrast ratio

### ISSUE THREAD

**Issue by varshaasj** [CMU-313/Mayan-EDMS#150]
Observed a score of 89% under the accessibility feature and the warning is the "Low-contrast text is difficult or impossible for many users to read." This is observed with the "search" button that has a low contrast ratio. And "view details" button has the same issue.
